bittensor/subtensor.py

```diff
@@ -496,7 +534,7 @@ def nominate(
 
     def delegate(
         self,
-        wallet: "bittensor.wallet",
+        wallet: bittensor.wallet,
```

Comment:
This probably doesn't matter here, but don't we tend to use quotes around bittensor typing to avoid circular imports?

Reply:
As we decided in [RFC 003: Bittensor Coding Standards](https://docs.google.com/document/d/1ERaRsZY-SFVJY0a179_rX2-8Ff-aFQWTUNn-gXj3UiI/edit#heading=h.cec1zrvrfmzv) -> Approach -> Typing :: `Prefer imported types for annotations; if not possible, use type names in quotes (for example to avoid circular imports).`
In this case there are no circular imports.